### ISSUE TITLE
Staging Review finding: Links to resources open in a new tab unexpectedly

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#94521]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** Identity
**Product:** Sign in page (and sign in modal)
**Feature:** Sign in Transition

## Findings details
**VA.gov Experience Standard - issue:** [User experiences an unexpected interaction with a component or pattern.](https://design.va.gov/about/experience-standards/#user-experiences-predictable-interactions-with-components)
**VA.gov Experience Standard - category:** [Usability](https://design.va.gov/about/experience-standards/#category-usability)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @erinrwhite (IA), @briandeconinck (Accessibility)

### Description

On both the modal and the sign-in page the links to resources and support pages open in a new tab without notice. Occurs with these links: Learn more about creating a Login.gov or ID.me account, signing in, verifying your identity. Whenever links open in a new tab, users should be notified using "(opens in a new tab)" within the link text.

Each of these links currently has an `aria-label` that includes "opens in a new tab," but that warning is valuable to all users and not just screen reader users.

## Recommended action

Our [guidance on opening in a new tab](https://design.va.gov/components/link/#behavior) says "open in a new tab only if clicking the link will result in the user losing progress or data. This scenario should be avoided when possible." 

Sign-in is a high-use function so I leave it to your team to interpret whether opening in a new tab is necessary. If this determination stands, please add “(opens in new tab)” to the link text to notify users when a link is opening in a new tab per our [content guidance](https://design.va.gov/content-style-guide/links#linking-to-external-sites).

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** N/A
- **Modality:** N/A
- **Design System Component:** [Link ](https://design.va.gov/components/language-toggle)
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** [Links](https://design.va.gov/content-style-guide/links)
- **Context:** User flow

---

### Next Steps for the VFS Team

- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue, comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
